static/js/pages/instances/InstanceList.tsx

```diff
@@ -36,6 +36,13 @@ const InstanceList: FC = () => {
   const [status, setStatus] = useState<string>("any");
   const [type, setType] = useState<string>("any");
 
+  const { setSharedNotify } = useSharedNotify();
+  useEffect(() => {
+    if (setSharedNotify) {
+      setSharedNotify(notify);
+    }
+  }, [setSharedNotify]);
```

Comment:
I forgot, this useEffect also depends on notify, we should add that to the array of dependencies.